pkg/testutils/manifest.go

```diff
@@ -588,3 +589,17 @@ func CredSecretLabelSelector(labelValue string) *apimetav1.LabelSelector {
 		MatchLabels: map[string]string{OCIRegistryCredLabelKeyForTest: labelValue},
 	}
 }
+
+func ManifestVersionIsCorrect(ctx context.Context, clnt client.Client, kymaName,
+	kymaNamespace,
+	moduleName, version string,
+) error {
```

Comment:
is this formatting intended? Looks a bit odd

Reply:
That's the result of gofumpt